tm2/pkg/bft/consensus/replay_test.go

```diff
@@ -1144,13 +1153,9 @@ func TestHandshakeUpdatesValidators(t *testing.T) {
 	genDoc, _ := sm.MakeGenesisDocFromFile(genesisFile)
 	handshaker := NewHandshaker(stateDB, state, store, genDoc)
 	proxyApp := appconn.NewAppConns(clientCreator)
-	if err := proxyApp.Start(); err != nil {
-		t.Fatalf("Error starting proxy app connections: %v", err)
-	}
-	defer proxyApp.Stop()
-	if err := handshaker.Handshake(proxyApp); err != nil {
-		t.Fatalf("Error on abci handshake: %v", err)
-	}
+	require.NoError(t, proxyApp.Start(), "Error starting proxy app connections")
+	defer func() { require.NoError(t, proxyApp.Stop()) }()
```

Comment:
```suggestion
       t.Cleanup(func() {
	    require.NoError(t, proxyApp.Stop())
	})
```

Reply:
done in 87fe7d8b6f2f145dc4ef2cecb6577d15ab40d3ba